add_npm_package.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Run as ./add_npm_package.sh name version strategy
+program_exists() {
+  which "$@" &> /dev/null;
+}
+
+if !(program_exists crudini); then
+  echo "crudini is not installed - installing..."
+  sudo yum install crudini
+  echo "crudini installed..."
+fi
+
+if !(program_exists npm2rpm); then
+  echo "npm2rpm is not installed - installing..."
+  sudo npm install -g npm2rpm
```

Comment:
ditto, or preferably use a non-global installation and add node_modules to .gitignore